modules/OFConnectionManager2/module/src/cxn_instance.c

```diff
@@ -2538,21 +2551,33 @@ ind_cxn_unblock_async_op(connection_t *cxn)
     }
 }
 
+void
+ind_unblock_async_op() {
+    if (former_async_pending_cnt > 0) {
+        former_async_pending_cnt--;
+    } else {
+        AIM_LOG_WARN("%s: former_async_pending_cnt already is 0", __FUNCTION__);
```

Comment:
maybe we should just AIM_DIE instead, since this should never happen and we want to make sure we know if it does.